src/test/java/uk/gov/hmcts/reform/laubackend/idam/feign/FeignErrorDecoderTest.java

```diff
@@ -72,6 +72,22 @@ void shouldReturnRetryableExceptionForGetRequestWithDetailsUrlAndStatus503() {
         assertThat(ex).isInstanceOf(RetryableException.class);
     }
 
+    @Test
```

Comment:
Both tests are testing for the same HTTP response, even though the tests name suggests otherwise

Reply:
comments incorporated